scripts/make_xcframework.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/zsh
+
+set -euo pipefail
+
+readonly BUILD_DIR="build"
+readonly OUTPUT_DIR="${BUILD_DIR}/output"
+
+clone() {
+    readonly VERSION="${1}"
+    git clone --branch "${VERSION}" --depth 1 https://github.com/krzyzanowskim/OpenSSL.git "${BUILD_DIR}/${VERSION}"
+}
+
+build_xcframework() {
+    readonly VERSION="${1}"
+    readonly OPENSSL_OUTPUT_DIR="${BUILD_DIR}/${VERSION}/Frameworks"
+
+    # Clean the openssl output directory (they have it in source control)
+    rm -rf "${OPENSSL_OUTPUT_DIR}"
+    mkdir -p "${OPENSSL_OUTPUT_DIR}"
+
+    # Create xcframework using the openssl script
+    (cd "${BUILD_DIR}/${VERSION}" && sh "scripts/create-frameworks.sh")
```

Comment:
They specify `bash` in [shebang](https://github.com/krzyzanowskim/OpenSSL/blob/1.1.1100/scripts/create-frameworks.sh#L1)

Reply:
Fixed